### ISSUE TITLE
Bug Report: [Sounds don't get recognized if they're contained within a folder, and defaults back to the regular soundbank.]

### ISSUE THREAD

**Issue by tivvo** [FNF-CNE-Devs/CodenameEngine#350]
### Which target are you using?
[X] Windows
[ ] Linux
[X] Mac

### Explain your issue
tl;dr if you have a sound inside of your sound folder and its inside of a folder within the sound folder it doesn't get recognized and in the odd specific case that theres only one sound it just defaults to the regular
##### Please check first if your issue haven't already been reported yet.
whatever
##### Re-Creation Steps
- create your sounds folder within the mod/sounds
- add a new folder inside then add the sound inside
- create a new script to play the sound, it should then default back to the fnf sound folder
- dats it, script should error out

(*unfixed, causes defaulting*)
![tiv@15-07-2024- explorer #Gnu](https://github.com/user-attachments/assets/34928049-9f90-44c9-88ce-35b1fd5f5cfb)
(*fixed version*)
![tiv@15-07-2024- explorer #Skink](https://github.com/user-attachments/assets/04ba5904-a246-486a-96d4-cbdd599fd845)

**Comment by tivvo** [comment]
tested out on mac also cuz i was trying to figure this out last night

**Comment by tivvo** [comment]
uh yeah i'll try to find the original code i did then compile a mod in a zip and send it here tomorrow when i get to my pc, getting to the other issue i'll do now since i can just do it in a sec

**Comment by NexIsDumb** [comment]
thanks!
